train.py

```diff
@@ -124,51 +141,51 @@ def main():
     run_metadata = tf.RunMetadata()
     summaries = tf.merge_all_summaries()
 
+    # Set up session
+    sess = tf.Session(config=tf.ConfigProto(log_device_placement=False))
     init = tf.initialize_all_variables()
     sess.run(init)
 
-    # Load and enqueue examples in the background.
-    coord = tf.train.Coordinator()
-    qr = tf.train.QueueRunner(queue, [enqueue])
-    qr.create_threads(sess, coord=coord, start=True)
-    threads = tf.train.start_queue_runners(sess=sess, coord=coord)
+    # start the tensorflow QueueRunner's
+    tf.train.start_queue_runners(sess=sess)
```

Comment:
I think we might still want to use a coordinator here to shut down all threads in a clean way.
We can even pass the coordinator to our custom thread runner and query the `coord.should_stop()` method to find out whether we should stop it.